doc/DEVELOPERS.md

```diff
@@ -102,3 +223,13 @@ You should see
  0.1.0
 (1 row)
 ```
+
+If you encounter any errors, you can check the log to find the reason. The log's location can be found in the `postgresql.conf` file.
+
+``` ini
+# These are only used if logging_collector is on:
+log_directory = 'log'                   # directory where log files are written,
+                                        # can be absolute or relative to PGDATA
+log_filename = 'postgresql-%a.log'      # log file name pattern,
```

Comment:
Remove `%a`

Reply:
For Fedora, the original file includes `%a`.

<img width="785" alt="Screenshot 2024-05-31 at 6 54 44 PM" src="https://github.com/pgmoneta/pgmoneta_ext/assets/113565433/260b6264-9f9f-4de8-a1a4-3ee519eaaaa0">